### ISSUE TITLE
minor typo in zrep_sync

### ISSUE THREAD

**Issue by onlime** [bolthole/zrep#1]
Please fix this little typo in zrep_sync, line 402:

```diff
-         if [[ "$MBUFFER != "" ]] ; then
+         if [[ "$MBUFFER" != "" ]] ; then
```

Thanks!

**Comment by ppbrown** [comment]
wow. how odd that got missed, sorry.
Pushed to git now.

**Comment by onlime** [comment]
Well, really, that's what I was wondering as well! :)
Thanks for the rapid fix.
fixed in 6d22592d32f2a7d635ac6560f6593c3ecce3be7f
